src/site/stages/build/process-cms-exports/transformers/paragraph-alert.js

```diff
@@ -8,7 +8,6 @@ const transform = entity => {
     fieldAlertBlockReference,
   } = entity;
   return {
-    contentModelType: entity.contentModelType,
```

Comment:
... instead of leaving it here?  Is it for clarity reasons or is there some other edge case?

Reply:
Because I forgot to add it in one of my transformers. The less we have to remember the better.

That said, I don't _actually_ know how useful it'll be after the transformation step. Maybe just debugging. Maybe not even that.

Reply:
Oops - I didn't read the filename and though this was something different.  This is what I thought I was talking about:

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/transform.js#L65